### DISCUSSION TITLE
Version Teleport most stable

### DISCUSSION THREAD

**Discussion by ncthi** [gravitational/teleport#49570] [Q&A]
Which version of teleport is the most stable?

**Comment by stevenGravy** [comment]
The most recent version is 17.0.2 as of responding to this. Teleport provides this production readiness information. Most customers and the Teleport Enterprise Cloud update to the first minor release after a major release (X.1.0).  v16 is at v16.4.8 which is above that minor release version.  See this page as well for expected upcoming releases.

https://goteleport.com/docs/upcoming-releases/#production-readiness
